net/eds-bootstrap-enroll/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2019 NXP
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=eds-bootstrap-enroll
+PKG_VERSION:=lsdk-1903
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/NXP/qoriq-eds-bootstrap.git
+PKG_SOURCE_VERSION:=385ec26acec4e1dc891c1d0e63ed577485a85f46
```

Comment:
Once the tag is fixed, this can be replaced with PKG_SOURCE_VERSION:=$(PKG_VERSION)

Reply:
Thanks. I will use the tag name LSDK-19.03.